lightning/src/chain/channelmonitor.rs

```diff
@@ -1187,8 +1181,6 @@ impl<Signer: Sign> ChannelMonitor<Signer> {
 
 	/// Get the list of HTLCs who's status has been updated on chain. This should be called by
 	/// ChannelManager via [`chain::Watch::release_pending_monitor_events`].
```

Comment:
Are parentheses needed here or are they optional? I noticed that you add them elsewhere.

Reply:
I believe parentheses are only required when the link is ambiguous in some way. cargo doc fails with an ambiguity error in that case.